src/dispatch/static/dispatch/src/components/Tiptap.vue

```diff
@@ -0,0 +1,104 @@
+<template>
```

Comment:
See above comment:
- Confusing name
- I would have expected the caller to do the save (it knows which value change) instead of the component itself.